### PR TITLE
Check all python version installed in Local machine

### LINKED ISSUE #2435: How to check Python version if more than one python version installed.
### Description

> A brief description of the Documentation Improvement or New Section request.
- In Prerequisites section of Scancode-Toolkit docs there should be some command describing that how one can check the 
  python version if they have more than one python version installed in there local machine.
- Example: _Python 2.7_ 
                  _Python 3.6_
                  _Python 3.9_
 ### Link to Documentation Page

> Where the confusion/inconsistency/incomplete documentation is.
- In Scancode-Toolkit Documents [Prerequisites](https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#before-installing).

### Select Category

-  [ ] Inconsistency
-  [x] New Section Request
-  [x] General Improvement
-  [ ] Typo/Mistakes
-  [ ] Other

### PR DESCRIPTION
**Pull request by s4-2**
Fixes #2435

### Tasks

* [ ] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [ ] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

File: docs/source/getting-started/install.rst
Comment (pombredanne) [inline docs/source/getting-started/install.rst]:
```suggestion
      If you have more than one installed Python version (e.g.: Python 3.6, 3.9, etc.) on Windows
```

File: docs/source/getting-started/install.rst
Comment (pombredanne) [inline docs/source/getting-started/install.rst]:
```suggestion
      run this to get a list of these versions::
```

File: docs/source/getting-started/install.rst
Comment (pombredanne) [inline docs/source/getting-started/install.rst]: Are you sure that this will always be a  `C:\Users>` prompt?

**Review by pombredanne — CHANGES_REQUESTED** [review]
Thank you! Please see my comments for your consideration.

File: docs/source/getting-started/install.rst
Comment (s4-2) [reply docs/source/getting-started/install.rst]: @pombredanne I tried running this command as C:\Users\Sadhana>py -0 it give the same result. Please check this screenshot
image

Comment (s4-2) [reply docs/source/getting-started/install.rst]: And by running this command _C:\Users>py -0p_ it gives the version+path
image

Comment (pratik0316) [reply docs/source/getting-started/install.rst]: The area where you are changing is the general section in docs and if you are changing that for Windows I think you should also include for other os.
apart from  C:\Users too, we can get the py versions from other directories ,by following the command  :)

**Comment by pombredanne** [comment]
Do you think you could squash your commits in one with a proper DCO signoff?
Thanks!

File: docs/source/getting-started/install.rst
Comment (s4-2) [reply docs/source/getting-started/install.rst]: Hi @Pratikrocks , sorry for the late response, if I am not getting wrong so you are saying that I should also add commands to know multiple python version in other OS like Mac and Linux, right?

Comment (pratik0316) [reply docs/source/getting-started/install.rst]: Yes, since it's a general section.
And moreover I think prefixing the directory for checking py versions in windows is not appropriate as we can access it from ay directories too :)

Comment (s4-2) [reply docs/source/getting-started/install.rst]: @Pratikrocks , so I C:\Users\User_Name>py -0 will be correct?

Comment (pratik0316) [reply docs/source/getting-started/install.rst]: Hmm not really , as we can run your command from d drive / f. Drive too right?

Comment (s4-2) [reply docs/source/getting-started/install.rst]: @Pratikrocks , I think the command is only supported where your python file is store. As in my case I have stored python in C drive.

**Comment by s4-2** [comment]
@pombredanne  Yes, sure! 
I am trying to do that since I have not use DCO signoff before so I will learn that as soon as possible.
Thank you

File: docs/source/getting-started/install.rst
Comment (pratik0316) [reply docs/source/getting-started/install.rst]: Oh okay cool 👍

**Comment by pombredanne** [comment]
@s4-2 do you mind to squash these in a single commit with your signoff? Thanks!

**Comment by s4-2** [comment]
@pombredanne , I am so sorry I forgot to to inform you that I am facing some corrupted file problem whenever I tried to store scancode toolkit in my laptop

**Comment by s4-2** [comment]
I checked my laptop and they say that this file is creating some issue with your OS but not yet confirmed so I tried again and clone this repository but same problem arised

**Comment by s4-2** [comment]
So I am requesting you to please assign someone who can finish this PR. I will now also trying to find out what is the issue with my laptop.

**Comment by s4-2** [comment]
Again sorry for late response.

**Comment by pombredanne** [comment]
@s4-2 I will take of adding your signoff before I merge. What I just need from you as a comment here is something such as this line, but using your name and email:

` Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>`

**Comment by pombredanne** [comment]
Closing as I will not  merge without a DCO Signed-off-by. Feel free to add and reopen! Thank you for your efforts anyway.

**Pull request closed without merging**
